src/components/about/FourPillTitle.tsx

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+const Heading = () => {
+  return (
+    <p className="pr-6 text-right font-serif text-3xl font-semibold text-csa-red-100">
```

Comment:
```suggestion
    <p className="pr-6 text-3xl font-semibold text-csa-red-100">
```
remove these classes, we won't be using text-right (we'll be using flexbox -- check out my prev comment) and serif isn't the right font, we'll add the fonts later!